### DISCUSSION TITLE
I'ts possible to set the full url from Youtube instead of the video Id on the Built-in video component?

### DISCUSSION THREAD

**Discussion by OrlandoP97** [GrapesJS/grapesjs#4155] [Q&A]
Hey, i need to add videos from youtube with the full URL,  i don't know if this is actually possible with the Built-in video component or i need to modify the source code, any help would be great, thanks

**Comment by artf** [comment]
You don't need to modify the source code, you can simply [update the video component](https://grapesjs.com/docs/modules/Components.html#update-component-type).
